h/search/index.py

```diff
@@ -89,23 +81,6 @@ def _prepare(self, annotation):
         return {self.op_type: operation}, data
 
 
-def _all_annotations(session, windowsize=2000):
-    # This is using a windowed query for loading all annotations in batches.
-    # It is the most performant way of loading a big set of records from
-    # the database while still supporting eagerloading of associated
-    # document data.
-    windows = column_windows(
```

Comment:
This happens to be the only usage of `column_windows` which gets in the way of upgrading SQLA.